Lombiq.NodeJs.Extensions/scripts/copy-assets.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @summary A script to copy files around; expects configuration in assets-to-copy.json or package.json (assetsToCopy).
+ * @description This script is a wrapper around the npm package "copyfiles". It reads its configuration from a file in
+ *              the consuming project named "assets-to-copy.json" or the "assetsToCopy" node in package.json.
+ */
+const { access } = require('node:fs/promises');
```

Comment:
I got complaints from Zoltán when I used `node:fs/promises`, because it's apparently not in v14. You can use `require('fs').promises` instead (like I did [in this commit](https://github.com/Lombiq/NodeJs-Extensions/commit/82a020ab056b96afb383cda4470ef9a27d72f7c3)) so let's use that for now. (same for all other `node:fs/promises` uses)